src/components/ProductGrid.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ProductDetail from './ProductDetail';
+
+function ProductGrid(props){
```

Comment:
Good job organizing things into another component. This probably makes more sense.